docs/semgrep-ci/configuring-blocking-and-errors-in-ci.md

```diff
@@ -3,59 +3,63 @@ slug: configuring-blocking-and-errors-in-ci
 append_help_link: true
 description: "Configure how Semgrep in CI pipelines handles errors and blocks findings."
 tags:
-    - Deployment
+ - Deployment
 title: Configure blocking findings
 hide_title: true
 ---
 
 import BlockFindingsErrorsConfigs from "/src/components/reference/_block-findings-errors-configs.mdx"
 
-# Configure blocking findings and errors
+# Configure handling of blocking findings and errors
```

Comment:
```suggestion
# Handling blocking findings and errors
```
Feel free to ignore this suggestion because I am not sure it is any better, having an `-ing` verb and `-ing` adjective.

Reply:
I do think it is a bit clearer!